app/controllers/guest_controller.rb

```diff
@@ -0,0 +1,5 @@
+class GuestController < ApplicationRecord 
+  def show 
+    guest=Guest.find(params[:id])
```

Comment:
You'll probably want guest to be an instance var so it's available in the view, so it would be `@guest`